pallets/connectors/src/lib.rs

```diff
@@ -224,36 +226,54 @@ pub mod pallet {
 	#[pallet::error]
 	pub enum Error<T> {
 		/// Failed to map the asset to the corresponding Connector's General
-		/// Index representation
+		/// Index representation and thus cannot be used as an
+		/// ivnestment currency.
 		AssetNotFound,
-		/// A pool could not be found
+		/// The metadata of the given asset does not declare it as a pool
+		/// currency and thus it cannot be used as an investment currency.
+		AssetMetadataNotPoolCurrency,
+		/// The given asset does not match the currency of the pool.
+		AssetNotPoolCurrency,
+		/// A pool could not be found.
 		PoolNotFound,
-		/// A tranche could not be found
+		/// A tranche could not be found.
```

Comment:
I have the idea the convention with rust docs is to have a `.` at the end of a one liner